docker-compose.yaml

```diff
@@ -56,6 +56,9 @@ x-airflow-common:
     AIRFLOW__CORE__LOAD_EXAMPLES: 'false' # do not load example DAGs
     AIRFLOW__CORE__DAGS_FOLDER: '/opt/dlme_airflow/dags'
     AIRFLOW__API__AUTH_BACKEND: 'airflow.api.auth.backend.basic_auth'
+    AIRFLOW_VAR_DATA_MANAGER_EMAIL: 'amcollie@stanford.edu'
```

Comment:
Do we have a more general email to use for the DLME project? (Maybe a follow-up ticket to pull this value from a config)

Reply:
Yes, this will get updated. It should either but a list (which I believe we have) or just @jacobthill for now. We'll finalize that.